docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4544,7 +4564,21 @@ four replicas of an ARP packet. These replicas will appear in the egress
 pipeline as independent packets with egress port set to PSA device port numbers
 corresponding to SDN port numbers 5, 12, 18 and 24. For more discussion on the
 translation between SDN ports and PSA device ports, refer to the
-[PSA Metadata Translation](#sec-translation-of-port-numbers) section.
+[PSA Metadata Translation](#sec-translation-of-port-numbers) section. Each
+replica can optionally have a list of backup replicas used as fallback ports
+for multicast. When the primary port (replica) goes down, the system
+uses the first backup replica for which its port is up. Whenever the primary
+port goes back up, the system will use the primary port again. In the case
+when the primary replica's ports and the backup replicas' ports are down,
+nothing is done for recovery until the primary replica or the backup replicas's
+ports go back up. When the primary and all the backups are down for a
+particular replica, the packet processing side effects can result in one of the
```

Comment:
What do you mean by side-effects? (I can make some guesses.). It might be helpful to mention an example. I don't think side-effects has been mentioned in the spec before.

Reply:
I agree it would be good to be explicit in the spec if it is not clear.  Side effects like counter updates, meter updates, and/or P4 register array writes in the P4 code after multicast replication is done.

Reply:
I have specified the packet processing side effects that Andy pointed out. Marking this comment as resolved.